nonce.go

```diff
@@ -72,10 +72,13 @@ func nonceForChunkSecretBox(i encryptionBlockNumber) Nonce {
 // above. There's almost certainly no harm in using the same nonces here as
 // above, since the encryption keys are ephemeral and the signatures already
 // have their own context, but at the same time it's a good practice.
-func nonceForChunkSigncryption(i encryptionBlockNumber) Nonce {
+func nonceForChunkSigncryption(i encryptionBlockNumber, isFinal bool) Nonce {
 	var n Nonce
-	copyEqualSizeStr(n[0:16], "saltpack_ploadsc")
-	binary.BigEndian.PutUint64(n[16:], uint64(i))
+	copyEqualSizeStr(n[0:15], "saltpack_ploads")
+	binary.BigEndian.PutUint64(n[15:], uint64(i))
```

Comment:
It feels cleaner to me to make the last 8 bytes the counter, though I guess it doesn't matter? Your call again.